.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,69 @@
+      render: bash
+      placeholder: |
+        Platform: (eg., Linux, Windows, etc)
+        Node version: v24.11.0
+        Platform: win32
+        Arch: x64
+        V8 version: 13.6.233.10-node.28
+        npm version: 11.6.1
+    validations:
+      required: true
+
+    validations:
```

YAMLlint (1.38.0)

[error] duplication of key "validations" in mapping

(key-duplicates)